pkg/pipelines/component/component/component.go

```diff
@@ -0,0 +1,40 @@
+package pipelines
+
+import (
+	"fmt"
+
+	"github.com/redhat-developer/gitops-generator/api/v1alpha1"
+	gitops "github.com/redhat-developer/gitops-generator/pkg"
+	"github.com/redhat-developer/kam/pkg/pipelines/ioutils"
+	"github.com/spf13/afero"
+)
+
+type CompomemtParameters struct {
```

Comment:
Hey @JainamShah-08 , one more thing...check the spelling here.

Reply:
Okay. I will make that changes.

Reply:
Like all the structs in one types file?